packages/x-charts/src/locales/ptBR.ts

```diff
@@ -17,104 +17,104 @@ export const ptBRLocaleText: Partial<ChartsLocaleText> = {
-  // chartConfigurationSeriesAxisReverse: 'Reverse series axis',
-  // chartConfigurationTooltipPlacement: 'Placement',
-  // chartConfigurationTooltipTrigger: 'Trigger',
-  // chartConfigurationLegendPosition: 'Position',
-  // chartConfigurationLegendDirection: 'Direction',
-  // chartConfigurationBarLabels: 'Bar labels',
-  // chartConfigurationColumnLabels: 'Column labels',
-  // chartConfigurationInterpolation: 'Interpolation',
-  // chartConfigurationSectionTooltip: 'Tooltip',
-  // chartConfigurationSectionLegend: 'Legend',
-  // chartConfigurationSectionLines: 'Lines',
-  // chartConfigurationSectionAreas: 'Areas',
-  // chartConfigurationSectionArcs: 'Arcs',
-  // chartConfigurationPaddingAngle: 'Padding angle',
-  // chartConfigurationCornerRadius: 'Corner radius',
-  // chartConfigurationArcLabels: 'Arc labels',
-  // chartConfigurationStartAngle: 'Start angle',
-  // chartConfigurationEndAngle: 'End angle',
-  // chartConfigurationPieTooltipTrigger: 'Trigger',
-  // chartConfigurationPieLegendPosition: 'Position',
-  // chartConfigurationPieLegendDirection: 'Direction',
+  chartTypeBar: 'Barra',
+  chartTypeColumn: 'Coluna',
+  chartTypeLine: 'Linha',
+  chartTypeArea: 'Área',
+  chartTypePie: 'Pizza',
+  chartPaletteLabel: 'Paleta de cores',
+  chartPaletteNameRainbowSurge: 'Onda de Arco-íris',
+  chartPaletteNameBlueberryTwilight: 'Crepúsculo de Blueberry',
+  chartPaletteNameMangoFusion: 'Fusão de Manga',
+  chartPaletteNameCheerfulFiesta: 'Festa Alegre',
+  chartPaletteNameStrawberrySky: 'Céu de Morango',
+  chartPaletteNameBlue: 'Azul',
+  chartPaletteNameGreen: 'Verde',
+  chartPaletteNamePurple: 'Roxo',
+  chartPaletteNameRed: 'Vermelho',
+  chartPaletteNameOrange: 'Laranja',
+  chartPaletteNameYellow: 'Amarelo',
+  chartPaletteNameCyan: 'Ciano',
+  chartPaletteNamePink: 'Rosa',
+  chartConfigurationSectionChart: 'Gráfico',
+  chartConfigurationSectionColumns: 'Colunas',
+  chartConfigurationSectionBars: 'Barras',
+  chartConfigurationSectionAxes: 'Eixos',
+  chartConfigurationGrid: 'Grade',
+  chartConfigurationBorderRadius: 'Raio de borda',
+  chartConfigurationCategoryGapRatio: 'Proporção de espaço entre categorias',
+  chartConfigurationBarGapRatio: 'Proporção de espaço entre séries',
+  chartConfigurationStacked: 'Empilhado',
+  chartConfigurationShowToolbar: 'Mostrar barra de ferramentas',
+  chartConfigurationSkipAnimation: 'Ignorar animação',
+  chartConfigurationInnerRadius: 'Raio interno',
+  chartConfigurationOuterRadius: 'Raio externo',
+  chartConfigurationColors: 'Cores',
+  chartConfigurationHideLegend: 'Ocultar legenda',
+  chartConfigurationShowMark: 'Mostrar marcação',
+  chartConfigurationHeight: 'Altura',
+  chartConfigurationWidth: 'Largura',
+  chartConfigurationSeriesGap: 'Espaçamento entre séries',
+  chartConfigurationTickPlacement: 'Posição da marcação',
+  chartConfigurationTickLabelPlacement: 'Posição do rótulo da marcação',
+  chartConfigurationCategoriesAxisLabel: 'Rótulo do eixo de categorias',
```

Comment:
I find "rótulo" an odd word to use in this case, not sure we do that ofter... 

I could find "título" or "texto" that could work better, but I'm not sure. 

WDYT? 🤔

Reply:
@djpremier & @bernardobelchior do you have any input here?

Reply:
I'd agree that "Título" would work better here, at least in the pt-PT variant